app/assets/stylesheets/user_index.scss

```diff
@@ -1,3 +1,26 @@
-.user-email {
-  display: none;
+.user {
+  background-color: $grey-light1;
+  margin: 10px;
```

Comment:
Properties should be ordered background-color, border, border-radius, box-shadow, margin, padding